Source/Scene/Expression.js

```diff
@@ -647,6 +662,8 @@ define([
             if (node._value === 'TILES3D_TILESET_TIME') {
                 node.evaluate = evaluateTime;
             }
+        } else if (node._type === ExpressionNodeType.TERNARY) {
+            node.evaluate = getEvaluateTernaryFunction(node._value);
```

Comment:
Place after binary. Also can you put the unary before binary in this function - not caused by you but it's good for consistency.